packages/server-wallet/src/__chain-test__/directly-funded-channel.test.ts

```diff
@@ -216,7 +215,21 @@ test.each(testCases)(
     const aBalanceFinal = await getBalance(aAddress);
     const bBalanceFinal = await getBalance(bAddress);
 
-    expect(BN.sub(aBalanceFinal, aBalanceInit)).toEqual(aFunding);
-    expect(BN.sub(bBalanceFinal, bBalanceInit)).toEqual(bFunding);
+    expect(BN.sub(aBalanceFinal, aBalanceInit)).toEqual(BN.from(1));
+    expect(BN.sub(bBalanceFinal, bBalanceInit)).toEqual(BN.from(4));
```

Comment:
The test values `(1,4)` are used a few times here and their meaning wasn't very obvious to me on reading their individual lines. The block in which they are used is not large, but it left me wondering whether the same values are inserted elsewhere as well.

Something like:
```
const finalOutcome = {
 aBalance: 1,
 bBalance: 4,
}
```
can make it easier to reason about the scope of those values. Similar for `initBalance`.

Reply:
Good idea! I've done something like that in https://github.com/statechannels/statechannels/pull/3582/commits/60029bdad9fa02b25ea298be1f3b952fcf921ff2